src/Innertube.ts

```diff
@@ -39,6 +41,7 @@ import type { ApiResponse } from './core/Actions.js';
 import { type IBrowseResponse, type IParsedResponse } from './parser/types/index.js';
 import type { INextRequest } from './types/index.js';
 import type { DownloadOptions, FormatOptions } from './types/FormatUtils.js';
+import {encodeReelSequence} from './proto/index.js';
```

Comment:
Please add spaces inside the brackets to match the code style used for the other imports, please address it in the other files too.

Reply:
Adapted and added a eslint rule to create warnings about this issue